src/autosuggest/internal.tsx

```diff
@@ -188,8 +186,8 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
     ...props,
     isEmpty,
     isFiltered,
-    recoveryText,
-    errorIconAriaLabel,
+    getRecoveryText: () => i18n('errorIconAriaLabel', restProps.errorIconAriaLabel),
+    getErrorIconAriaLabel: () => i18n('recoveryText', restProps.recoveryText),
```

Comment:
Make recovery text and error label lazy-loadable, only when this feature is used